_posts/2021-11-24-ghelp-stats.md

```diff
@@ -0,0 +1,35 @@
+---
+site: [ghelp, galaxy, statistics]
+title: 'An overview on the Galaxy Help Forum'
+tags: [GOATS]
+supporters:
+- galaxy-europe
+---
+
+As part of the Galaxy Community, you probably have visited the [Galaxy Help Forum](https://help.galaxyproject.org/). This website, together with the different Gitter channels, is one of the primary meeting places where Galaxy users turn to in search of aswers. Since November 2018, when it was put into service, more than 3.000 users have been registrated, and around 9.3000 posts have been written. Currently, the average number of daily visits is around 800 (fig. 1), having reached its peak in May 2021, with 2.800 visits in a single day.
```

Comment:
I think here is missing the aspect that every user is encouraged to also help others ... this is your message with the quote isn't it?